modules/gitops-release-notes-1-7-1.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+:_content-type: REFERENCE
+
+[id="gitops-release-notes-1-7-1_{context}"]
+= Release notes for {gitops-title} 1.7.1
+
+{gitops-title} 1.7.1 is now available on {product-title} 4.8, 4.9, 4.10, 4.11, and 4.12.
+
+[id="errata-updates-1-7-1_{context}"]
+== Errata updates
```

Comment:
This release only has security fixes, right? When we do a release that has more than just the Errata section, are we able to make sure the Errata goes last? 😊

Reply:
yes, @harrietgrace. Organization and placement of our content is in our hands. Since we did not have anything else for the GitOps 1.7.1 release, other than these security fixes, we had only one section of errata updates. In future releases, if we have other sections about new features, bug fixes, known issues, we would make sure to keep any errata updates in the end.